include/GPM/Shape3D/OrientedBox.hpp

```diff
@@ -0,0 +1,85 @@
+﻿/*
+ * Copyright (C) 2021 Amara Sami, Dallard Thomas, Nardone William, Six Jonathan
+ * This file is subject to the LGNU license terms in the LICENSE file
+ * found in the top-level directory of this distribution.
+ */
+
+#pragma once
+
+#include "Volume.hpp"
+#include "../Vector3.hpp"
+#include "../Transform.hpp"
+#include "../Referential.hpp"
+#include "AABB.hpp"
+
+namespace GPM
+{
+    class OrientedBox : public Volume
+    {
+        protected:
+
+        Referential m_referential;
+        float       m_iI {0.f}, m_iJ {0.f}, m_iK {0.f}; 
```

Comment:
I suggest we leave data members `public`

Reply:
I suggest changing `float m_iI{0.f}, m_iJ{0.f}, m_iK{0.f}` to more explicit names. I suggest `Vec3 bounds`